cmd/query/app/querysvc/v2/adjuster/hash.go

```diff
@@ -52,16 +55,27 @@ func (s *SpanHashDeduper) Adjust(traces ptrace.Traces) {
 			for k := 0; k < spans.Len(); k++ {
 				span := spans.At(k)
 				span.CopyTo(hashSpan)
-				h, err := s.computeHashCode(
-					hashTrace,
-				)
-				if err != nil {
-					jptrace.AddWarnings(span, fmt.Sprintf("failed to compute hash code: %v", err))
-					span.CopyTo(dedupedSpans.AppendEmpty())
-					continue
+
+				var spanHash int64
```

Comment:
this should be a helper function (or two) in `jptrace` package, similar to how you added them to `model/`